kms/keysource.go

```diff
@@ -233,6 +236,17 @@ func (c CredentialsProvider) ApplyToMasterKey(key *MasterKey) {
 	key.credentialsProvider = c.provider
 }
 
+// HTTPClient is a wrapper around http.Client used for configuring the
+// AWS KMS client.
+type HTTPClient struct {
```

Comment:
This appears to be missing a constructor, rendering it unusable for external consumers.

Reply:
lol good catch sorry :facepalm: